static/gsApp/components/quotaExceededNavItem.tsx

```diff
@@ -0,0 +1,292 @@
+import {Fragment, useCallback, useEffect, useState} from 'react';
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import {motion, type MotionProps} from 'framer-motion';
+
+import {promptsUpdate, usePromptsCheck} from 'sentry/actionCreators/prompts';
+import {Checkbox} from 'sentry/components/core/checkbox';
+import {IconWarning} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {DataCategory} from 'sentry/types/core';
+import type {Organization} from 'sentry/types/organization';
+import useApi from 'sentry/utils/useApi';
+import {usePrefersStackedNav} from 'sentry/views/nav/prefersStackedNav';
+import {SidebarButton} from 'sentry/views/nav/primary/components';
+import {
+  PrimaryButtonOverlay,
+  usePrimaryButtonOverlay,
+} from 'sentry/views/nav/primary/primaryButtonOverlay';
+
+import AddEventsCTA, {type EventType} from 'getsentry/components/addEventsCTA';
+import withSubscription from 'getsentry/components/withSubscription';
+import type {Subscription} from 'getsentry/types';
+import {getCategoryInfoFromPlural} from 'getsentry/utils/billing';
+import {listDisplayNames, sortCategoriesWithKeys} from 'getsentry/utils/dataCategory';
+
+function QuotaExceededContent({
+  exceededCategories,
+  subscription,
+  organization,
+  onDismiss,
+  isDismissed,
+}: {
+  exceededCategories: string[];
+  isDismissed: boolean;
+  onDismiss: () => void;
+  organization: Organization;
+  subscription: Subscription;
+}) {
+  const eventTypes: EventType[] = exceededCategories.map(category => {
+    const categoryInfo = getCategoryInfoFromPlural(category as DataCategory);
+    return (categoryInfo?.plural ?? category) as EventType;
+  });
+  return (
+    <Container>
+      <Header>
+        <HeaderTitle>{t('Status')}</HeaderTitle>
+      </Header>
+      <Body>
+        <Title>{t('Quota Exceeded')}</Title>
+        <Description>
+          {tct(
+            'You’ve run out of [exceededCategories] for this billing cycle. This means we are no longer monitoring or ingesting events and showing them in Sentry.',
+            {
+              exceededCategories: listDisplayNames({
+                plan: subscription.planDetails,
+                categories: exceededCategories,
+                hadCustomDynamicSampling: subscription.hadCustomDynamicSampling,
+              }),
+            }
+          )}
+        </Description>
+        <ActionContainer>
+          <AddEventsCTA
+            organization={organization}
+            subscription={subscription}
+            buttonProps={{
+              size: 'xs',
+            }}
+            eventTypes={eventTypes}
+            notificationType="overage_critical"
+            referrer={`overage-alert-sidebar-${eventTypes.join('-')}`}
+            source="quota-overage"
+            handleRequestSent={() => {}}
+          />
+          <DismissContainer>
+            <Checkbox
```

Comment:
Would it make more sense for this to be a button? A checkbox that immediately dismisses doesn't make sense to me

Reply:
This comes straight from [design](https://www.figma.com/design/1zN1dG3sO1wER0S9KnwL6P/Quota-Notification?node-id=0-1&p=f&t=5fS8k5E8A4UE5Aa8-0) but definitely something I ask the team about (ie. if we're hardset on a checkbox)